docs/topics/eclipse-about-plugin.adoc

```diff
@@ -1,10 +1,15 @@
 // Module included in the following assemblies:
 //
 // * docs/getting-started-guide/master.adoc
+// * docs/eclipse-code-ready-studio-guide/master.adoc
 
 [id="eclipse-about-plugin_{context}"]
 = About the {PluginName}
 
 The {PluginName} for the {ProductName} provides assistance directly in Eclipse and Red Hat CodeReady Studio for developers making changes for a migration or modernization effort.
 
 The {PluginName} analyzes your projects using customizable rulesets, marks migration issues in the source code, provides guidance to fix the issues, and offers automatic code replacement, or Quick Fixes, if possible.
+
+ifdef::eclipse-code-ready-studio-guide[]
+A similar extension provides assistance in Visual Studio Code. For information about that extension, see the link:{ProductDocVscGuideURL}[_Visual Studio Code Extension Guide_]
```

Comment:
A bit verbose. I recommend this: "For information about a similar extension for Visual Studio Code, see the [link].